ras_backstage/controllers/error_decorator.py

```diff
@@ -21,7 +21,11 @@ def wrapper(*args, **kwargs):
             log.error(e.to_dict())
             raise
         except HTTPError as e:
-            raise RasError([str(e)])
+            try:
+                detail = e.response.json().get('detail', "No further detail")
+            except Exception:
+                detail = "No further detail"
+            raise RasError([str(e), detail])
         except Exception as e:
```

Comment:
What are your thoughts around catching 'time out' errors and handling them gracefully since this is the proxy to the angular app?

Reply:
Possibly, although I've no idea of the requirements for it. A timeout would simply be propagated as a 500 error with the relevant text in the json body. We could possibly introduce backoff/retry, although we have the same general problem across all of the RAS services.